ush/forecast_postdet.sh

```diff
@@ -425,7 +449,41 @@ WW3_nml() {
 
 WW3_out() {
   echo "SUB ${FUNCNAME[0]}: Copying output data for WW3"
-  # TODO: Need to add logic to copy restarts from DATArestart/WW3_RESTART to COMOUT_WAVE_RESTART
+
+  # Copy wave namelist from DATA to COMOUT_CONF after the forecast is run (and successfull)
+  ${NCP} "${DATA}/ww3_shel.nml" "${COMOUT_CONF}/ufs.ww3_shel.nml"
+
+  # Copy WW3 restarts at the end of the forecast segment to COM for RUN=gfs|gefs
+  if [[ "${COPY_FINAL_RESTARTS}" == "YES" ]]; then
+    local seconds source_file target_file
```

Comment:
There is no `source_file`, only a `restart_file`.
Also, no `seconds` are used here.
```suggestion
    local restart_file
```